shared/structures/src/members/OrganizationRecordsConfiguration.ts

```diff
@@ -51,6 +51,19 @@ export class FinancialSupportSettings extends AutoEncoder {
     @field({ decoder: StringDecoder, optional: true })
     warningText = FinancialSupportSettings.defaultWarningText
 
+    /**
+     * Whether a member can self assign financial support.
+     * If false a member who chooses financial support cannot self subscribe.
+     */
+    @field({ decoder: BooleanDecoder, optional: true, version: 308 })
+    isPreventSelfAssignment = false
```

Comment:
```suggestion
    preventSelfAssignment = false
```
Denk dat dit een betere naam is, nu vind ik het nogal lastig om te lezen in de code en is daardoor de betekenis niet altijd super duidelijk